### PR TITLE
Cynara_zkthon

### PR DESCRIPTION
**Pull request by Cynaral**
## Description

This pull request is for the zkEVM Challenge 1,2,3 . I have reviewed the zkEVM documentation and shared my understanding through a Twitter thread using the hashtags #zkThon and #zkEVM. The link to my Twitter thread is https://twitter.com/cynara2007/status/1640021799445438465?s=20

link to deploy: https://testnet-zkevm.polygonscan.com/tx/0x0f19c19226dbd51cecf9b31dba8bd1c33869e4810a8afd233c827f9b3a5cf9d9

## Checklist

- [x] I have created a new file named `solution-1.md, solution-2.md, solution-3.md` for this challenge.
- [x] I have added the details of my submission to the [Airtable form](https://airtable.com/shr21z0FfPImZfYBQ).
- [x] I have named my branch `Cynara_zkthon`.

### CODE REVIEW

File: solution-2.md
Comment (megabyte0x) [inline solution-2.md:1]: Please add the deployed contract link to the file.

Thanks 😄

**Review by megabyte0x — CHANGES_REQUESTED** [review]

File: solution-2.md
Comment (Cynaral) [reply solution-2.md:1]: ok) done

Comment (megabyte0x) [reply solution-2.md:1]: Good work 🔥

File: solution-3.md
Comment (megabyte0x) [inline solution-3.md:1]: Please add the interface code used to interact with the contract. 

Thanks

**Review by megabyte0x — CHANGES_REQUESTED** [review]

**Pull request closed without merging**